src/components/diffViewer/DiffViewer.tsx

```diff
@@ -0,0 +1,42 @@
+import {Diff2Html} from 'diff2html';
+import * as React from 'react';
+import * as unidiff from 'unidiff';
+import * as styles from './styles/DiffViewer.scss';
+
+export interface DiffViewerProps extends React.ClassAttributes<DiffViewer> {
+    first: string;
+    second: string;
+}
+
+export class DiffViewer extends React.Component<DiffViewerProps> {
+    static OutputFormat = {
+        Side: 'side-by-side',
+        Line: 'line-by-line',
+    };
+
+    static InputFormat = {
+        Diff: 'diff',
+        JSON: 'json',
+    };
+
+    static Matching = {
+        Lines: 'lines',
+        Words: 'words',
+        None: 'none',
+    };
+
+    render() {
+        const diff = unidiff.diffLines(this.props.first, this.props.second);
+        const formattedDiff = unidiff.formatLines(diff, {context: 3});
+        const html = Diff2Html.getPrettyHtml(formattedDiff, {
+            inputFormat: DiffViewer.InputFormat.Diff,
+            showFiles: false,
+            matching: DiffViewer.Matching.Words,
+            outputFormat: DiffViewer.OutputFormat.Line,

```

Comment:
I thought about it and for now, these parameters are mostly for the style. Changing them would break the current style. I created the variables just to show what the options are if we ever add features to it. I don't mind using props though if people don't mind breaking the style by using them.